shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_property_object_impl.cpp

```diff
@@ -172,6 +172,36 @@ ErrCode INTERFACE_FUNC TmsClientPropertyObjectBaseImpl<Impl>::setPropertyOrder(I
     return OPENDAQ_ERR_INVALID_OPERATION;
 }
 
+template <class Impl>
+ErrCode INTERFACE_FUNC TmsClientPropertyObjectBaseImpl<Impl>::beginUpdate()
+{
+    const auto beginUpdateId = clientContext->getReferenceBrowser()->getChildNodeId(nodeId, "BeginUpdate");
+
+    OpcUaCallMethodRequest request;
+    request->inputArgumentsSize = 0;
+    request->objectId = nodeId.getValue();
+    request->methodId = beginUpdateId.getValue();
+    const auto response = client->callMethod(request);
+    const auto success = response->statusCode == UA_STATUSCODE_GOOD;
+    
+    return (success) ? OPENDAQ_SUCCESS : OPENDAQ_ERR_GENERALERROR;
```

Comment:
I'm not 100% we should crash the client if this fails. The OPC UA client should be more forgiving.